app/ui-utils/client/lib/RoomHistoryManager.js

```diff
@@ -151,8 +189,10 @@ export const RoomHistoryManager = new class {
 			typeName = (curRoomDoc ? curRoomDoc.t : undefined) + (curRoomDoc ? curRoomDoc.name : undefined);
 		}
 
+		// const showMessageInMainThread = getUserPreference(Meteor.userId(), 'showMessageInMainThread', false);
```

Comment:
will this still be used?

Reply:
I will, just commented to let you fetch all messages instead.